roles/common/defaults/main.yml

```diff
@@ -2,4 +2,5 @@
 use_host_domain: false
 host_domain: novalocal
 selinux_policy: targeted
-selinux_state: permissive
\ No newline at end of file
+selinux_state: permissive
+distributive_rpm_url: "https://bintray.com/artifact/download/siddharthist/distributive-testing/distributive-0.2.4-1.x86_64.rpm"
```

Comment:
I would love it if anyone on our team with access could just download it from there and reupload it to CiscoCloud.